lutris/util/display.py

```diff
@@ -159,11 +158,9 @@ def change_resolution(resolution):
 
 def restore_gamma():
     """Restores gamma to a normal level."""
-    global XGAMMA_FOUND
-    if XGAMMA_FOUND is None:
-        XGAMMA_FOUND = system.find_executable('xgamma')
-    if XGAMMA_FOUND is True:
-        subprocess.Popen(["xgamma", "-gamma", "1.0"])
+    XGAMMA_PATH = system.find_executable('xgamma')
```

Comment:
Cool but this removes caching so we make a call to `which xgamma` every single time. And yes, this is way more costly than one would think.

Reply:
You're right. I hope it's better now.